src/components/fields/schemaFields/widgets/cssClassWidgets/utils.ts

```diff
@@ -142,12 +142,13 @@ export function calculateNextSpacing(
       .filter((rule) =>
         spacingUpdate.side == null ? rule.side == null : rule.side != null,
       )
-      .map(
-        (x) =>
-          `${prefix}${x.side ?? ""}-${x.size < 0 ? "n" : ""}${Math.abs(
-            x.size,
-          )}`,
-      ),
+      .map((x) => {
```

Comment:
[Nit]: I think this reads cleaner if you have defaults and clean up the types and while destructuring, and then just have the map logic in the map function:

```suggestion
      .map(
        ({ side = "", size = 0 }: { side: string, size: number }) =>
          `${prefix}${side}-${size < 0 ? "n" : ""}${Math.abs(size)}`,
      ),
```

Reply:
The values can be `null` so the default value during destructuring doesn't work. I can clean it up a bit though.